test/elixir-mode-font-test.el

```diff
@@ -125,7 +141,8 @@ end"
   (elixir-test-with-temp-buffer
       ":oriole
 :andale
-:ms2pid"
+:ms2pid
+:CapitalizedAtom"
```

Comment:
You didn't add any `should` checks for the newly added line so adding it didn't really add anything of value.